src/field_theory/ratfunc.lean

```diff
@@ -430,6 +443,86 @@ instance : comm_ring (ratfunc K) :=
   zsmul_neg' := λ _, by smul_tac,
   npow := npow_rec }
 
+variables {K}
+
+section lift_hom
+
+variables {G₀ L : Type*} [comm_group_with_zero G₀] [field L]
+
+/-- Lift a `polynomial K →*₀ G₀` to a `ratfunc K →*₀ G₀`
```

Comment:
```suggestion
/-- Lift an injective function `polynomial K →*₀ G₀` to a function `ratfunc K →*₀ G₀`
```
I stared twenty seconds at `polynomial K →*₀ G₀` before realizing this was a function and not a polynomial with coefficients in `K →*₀ G₀`, so it's better to be clear.